libraries/roomselect/impl/src/main/kotlin/io/element/android/libraries/roomselect/impl/RoomSelectView.kt

```diff
@@ -235,7 +235,8 @@ private fun RoomSummaryView(
             // Name
             Text(
                 style = ElementTheme.typography.fontBodyLgRegular,
-                text = summary.name,
+                // If name is null, we do not have space to render "No room name", so just use `#` here.
+                text = summary.name ?: "#",
```

Comment:
Given the designs, I thought there would be enough free space. Is this a problem only in German translations?

<img width="392" alt="image" src="https://github.com/element-hq/element-x-android/assets/480955/0c266074-272d-4539-8598-aa9ed8770af4">

Reply:
My bad, this is a mistake, will fix.

Reply:
https://github.com/element-hq/element-x-android/pull/2752/commits/8272b7a16c707e1c30f3f9daaa2a8ff0dc36b525